pkg/server/config.go

```diff
@@ -237,6 +237,12 @@ func GetTotalMemory() (int64, error) {
 			humanize.IBytes(mem.Total), humanize.Bytes(math.MaxInt64))
 	}
 	totalMem := int64(mem.Total)
+	checkTotal := func(x int64) error {
```

Comment:
nit: consider making the signature here `(int64, error)`. That way you can hold up the usual convention (that you return zero when there's an error) more explicitly.